files/pt-br/web/css/_doublecolon_selection/index.md

```diff
@@ -94,11 +94,7 @@ Facilite para os usuários ver e ouvir o conteúdo, incluindo a separação do p
 
 ## Especificações
 
-| Especificações                                                                | Status                            | Comentário         |
-| ----------------------------------------------------------------------------- | --------------------------------- | ------------------ |
-| {{SpecName('CSS4 Pseudo-Elements', '#selectordef-selection', '::selection')}} | {{Spec2('CSS4 Pseudo-Elements')}} | Definição inicial. |
-
-> **Nota:** `::selection` estava nos rascunhos do Nível 3 dos Seletores de CSS, mas foi removido na fase Recomendação do Candidato porque estava subespecificado (especialmente com elementos aninhados) e a interoperabilidade não foi alcançada ([com base na lista de discussão de estilos W3C](http://lists.w3.org/Archives/Public/www-style/2008Oct/0268.html)). Ele retornou no [Nível 4 dos Pseudo-Elementos](http://dev.w3.org/csswg/css-pseudo-4/).
```

Comment:
removed in [en-US](https://developer.mozilla.org/en-US/docs/Web/CSS/::selection#specifications)